tg/models/__init__.py

```diff
@@ -241,7 +241,22 @@ def update_msg_content(
             if msg["id"] != msg_id:
                 continue
             msg["content"] = message_content
-        return True
+            return True
+        return False
+
+    def update_msg_content_opened(self, chat_id: int, msg_id: int):
+        for msg in self.msgs[chat_id]:
+            if msg["id"] != msg_id:
+                continue
+            msg = MsgProxy(msg)
+            if msg.type == "voice":
+                msg.is_listened = True
+            elif msg.type == "recording":
+                msg.is_viewed = True
+            # TODO: start the TTL timer for self-destructing messages
+            # that is the last case to implement
+            # https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1update_message_content_opened.html
+            return
```

Comment:
redundant return

Reply:
why is it? without it, we will continue to loop over msgs until they end, it's useless ops

Reply:
oh sorry, it though it is outside of the loop.
then plz change to `break` for redability